blog/2024-04-07-deprecate_add_hass_job.md

```diff
@@ -0,0 +1,9 @@
+---
+author: J. Nick Koston
+authorURL: https://github.com/bdraco
+title: "Deprecating `async_add_hass_job`"
+---
+
+`async_add_hass_job` is deprecated and will be removed in Home Assistant 2025.5.
```

Comment:
Maybe add some short background why we do this? Eg it's part of the effort to increase performance and simplify the core job API.

Reply:
Good idea to add context.  It was a bit light on details.